style.css

```diff
@@ -0,0 +1,692 @@
+body {
+  font-family: Lato, sans-serif;
+  margin: 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background: #f8f8f8;
+}
+
+.wrapper {
+  padding: 0 15% 10%;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, kindly make sure your `Headline` contents on the `About` page are well centered on the page in order to match tye provided Figma design.

  - ### YOUR DESIGN
  ![image](https://github.com/obibaadoma/captone_DaaNkwa/assets/87197412/9132fa8b-b02e-4fd5-9076-5b960df4bf46)


  - ### BEHANCE DESIGN
  
  ![image](https://github.com/obibaadoma/captone_DaaNkwa/assets/87197412/6b5cfdd9-63fd-4dae-8a63-df8b2ddd4bc1)